config/keyboards/kanata.kbd

```diff
@@ -0,0 +1,33 @@
+(defcfg
+  linux-continue-if-no-devs-found yes ;; Workaround for GH-1156
+  concurrent-tap-hold yes
+)
+
+;; Required to specify keys which should be considered in tap-hold-press (=spc, tab for alt-spc, alt-tab)
+;; See https://github.com/jtroo/kanata/blob/fc850fab9da7d0bf377e215f0b923062e037ff64/docs/config.adoc?plain=1#L142-L143
+(defsrc caps lalt ralt spc tab f6 a c p)
+
+(defvar
+  tap-timeout  200
+  hold-timeout 200
+  tt $tap-timeout
+  ht $hold-timeout
+)
+
+(deflayermap (base)
+  ;; Avoiding tap-hold for caps to make stable behavior
+  caps lctl
+  lalt (tap-hold-press $tt $ht muhenkan lalt)
+  ralt (tap-hold-press $tt $ht henkan ralt)
+  f6 (layer-while-held help)
+)
+
+(deflayermap (help)
+)
+
+;; Escape hatch for use of caps. See GH-963.
+;; Should be disabled on default layer with the last argument. For example, if using "a" in the combination, "main" will be "mian"
+(defchordsv2
+  ;; Don't use caps for inputs combination
+  (c a p) caps 500 all-released (base)
+)
```

Comment:
Unnecessary since https://github.com/kachick/dotfiles/commit/a2c40276bbb20682232db72ceab9c2263eea40b7

Should be removed, or replaced to similar way of the both shift press.